frontend/src/metabase/reference/metrics/MetricDetail.jsx

```diff
@@ -25,14 +25,15 @@ import {
   getError,
   getLoading,
   getUser,
-  getIsEditing,
   getIsFormulaExpanded,
   getForeignKeys,
 } from "../selectors";
 
 import * as metadataActions from "metabase/redux/metadata";
 import * as actions from "metabase/reference/reference";
 
+import _ from "underscore";
```

Comment:
Not a big deal but our general convention is to group external library imports together at the top.